t/24-worker-settings.t

```diff
@@ -110,6 +110,22 @@ subtest 'instance-specific settings' => sub {
         },
         'global settings (instance 2)'
     ) or diag explain $settings2->global_settings;
+    my $settings3 = OpenQA::Worker::Settings->new(3);
+    is_deeply(
+        $settings3->global_settings,
+        {
+            CRITICAL_LOAD_AVG_THRESHOLD => 40,
+            GLOBAL => 'setting',
+            WORKER_HOSTNAME => '127.0.0.1',
+            WORKER_CLASS => 'qemu_aarch64',
+            LOG_LEVEL => 'test',
+            LOG_DIR => 'log/dir',
+	    PARALLEL_ONE_HOST_ONLY => 1,
+            RETRY_DELAY => 5,
+	    RETRY_DELAY_IF_WEBUI_BUSY => 60,
+        },
+        'global settings (instance 3)'
+    ) or diag explain $settings2->global_settings;
```

Comment:
```suggestion
```

I don't think we need that test. You haven't changed anything in the settings parser itself and we should not add such a big test for every setting we introduce. (Or are changes in the settings parser still going to be needed?)

Reply:
Done